Xamarin.Forms.Platform.iOS/Renderers/ButtonElementManager.cs

```diff
@@ -85,5 +85,11 @@ internal static void OnButtonTouchUpInside(IButtonController element)
 			element?.SendReleased();
 			element?.SendClicked();
 		}
+
+		public static void UpdateDisabledTextColor(Button button, UIButton nativeButton)
+		{
+			var title = new NSAttributedString(nativeButton.CurrentTitle, new UIStringAttributes { ForegroundColor = button.DisabledTextColor.ToUIColor() });
```

Comment:
`var title = new NSAttributedString(nativeButton.CurrentTitle ?? string.Empty, new UIStringAttributes { ForegroundColor = button.DisabledTextColor.ToUIColor() });`

This change fix the iOS issue:
<img width="409" alt="Captura de pantalla 2019-11-06 a las 9 26 25" src="https://user-images.githubusercontent.com/6755973/68281306-af471180-0077-11ea-9ab7-c982d4bc0b1d.png">